boards/feather-m0/include/board.h

```diff
@@ -59,9 +59,11 @@ extern "C" {
  * @{
  **/
 #define SX127X_PARAM_SPI                SPI_DEV(0)
+#if defined(BOARD_FEATHER_M0_LORA)
 #define SX127X_PARAM_SPI_NSS            GPIO_PIN(PA, 6)
 #define SX127X_PARAM_RESET              GPIO_PIN(PA, 8)
 #define SX127X_PARAM_DIO0               GPIO_PIN(PA, 9)
+#endif
 #define SX127X_PARAM_DIO1               GPIO_UNDEF
 #define SX127X_PARAM_DIO2               GPIO_UNDEF
 #define SX127X_PARAM_DIO3               GPIO_UNDEF
```

Comment:
Why not include the entire block in the `#ifdef`

Reply:
Because that's all I needed. `SX127X_PARAM_SPI` and `SX127X_PARAM_PASELECT` look invariant.
But it's true that DIO1-3 can be wired, this is also true for the `feather-m0-lora` variant, so what about
something like:

```
/**
 * @name    Configuration for Feather M0 LoRa and the SX1276 module
 * @{
 **/
#define SX127X_PARAM_SPI                SPI_DEV(0)

// These pins must be defined, in the lora variant they are already set, but
// with a FeatherWing they've to be wired and configured.
#if defined(BOARD_FEATHER_M0_LORA)
#define SX127X_PARAM_SPI_NSS            GPIO_PIN(PA, 6)
#define SX127X_PARAM_RESET              GPIO_PIN(PA, 8)
#define SX127X_PARAM_DIO0               GPIO_PIN(PA, 9)
#endif

// DIO1-3 are undefined by default, but they can be wired.
#ifndef SX127X_PARAM_DIO1
#define SX127X_PARAM_DIO1               GPIO_UNDEF
#endif
#ifndef SX127X_PARAM_DIO2
#define SX127X_PARAM_DIO2               GPIO_UNDEF
#endif
#ifndef SX127X_PARAM_DIO3
#define SX127X_PARAM_DIO3               GPIO_UNDEF
#endif

#define SX127X_PARAM_PASELECT           (SX127X_PA_BOOST)
/** @} */
```

Reply:
If it's wired externally it wouldn't have to be connected to SPI0, right? 
I'm also surprised `SX127X_PARAM_DIOx` are not `GPIO_UNDEF` - this seems to be an issue in `sx127x_params.h`

Reply:
> If it's wired externally it wouldn't have to be connected to SPI0, right?

From https://learn.adafruit.com/radio-featherwing/pinouts#spi-data-pins-fixed-2057617-2

> The three SPI data pins (MOSI/MISO/SCK) are hardwired to these three pads, which are use for the default SPI interface on all Feathers:

So I think `SPI_DEV(0)` is correct for all variants of `feather-m0`

Reply:
>  I'm also surprised SX127X_PARAM_DIOx are not GPIO_UNDEF - this seems to be an issue in sx127x_params.h

Yes.. the pins could be different for every board/shield no? Or is there a reasonable default?
This was implemented in commit 0ccc26453892ba78c5d30b8bfe3d20391ea8458a by @aabadie (ping)